### PR TITLE
Updating cli docs for octopus.migrator.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octopus.migrator.exe.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 52

### CODE REVIEW

File: docs/api-and-integration/octopus.migrator.exe-command-line/import.md [hunk 1]
```diff
@@ -10,24 +10,6 @@ The export must have been made from an Octopus Server running the same release v
 Usage:
 
 ```text
-Octopus.Migrator import [<options>]
```
Comment (wordlee) [inline docs/api-and-integration/octopus.migrator.exe-command-line/import.md]: It looks like this is broken again :(

Comment (matt-richardson) [reply docs/api-and-integration/octopus.migrator.exe-command-line/import.md]: Yeah, still borked. Just trying to get https://github.com/OctopusDeploy/OctopusDeploy/pull/3684 in.

Comment (matt-richardson) [reply docs/api-and-integration/octopus.migrator.exe-command-line/import.md]: Sorry - meant to add a comment on this PR.

**Comment by matt-richardson** [comment]
Closing - we're still trying to fix the migrator help cli.

**Pull request closed without merging**